src/server/markup.js

```diff
@@ -1,16 +1,20 @@
-export default (reactString, initialData, webserver) => {
+export default (reactString, initialData, webserver, config) => {
+  const stylesheet = process.env.NODE_ENV === 'production'
+    ? `<link rel='stylesheet' href='${webserver}/dist/styles.css'>`
+    : '';
```

Comment:
Is is a good practise to just do the logic here and or should the compilation of the stylesheet link be extracted as a `stylesheet(webserver)` function?